sdk/communication/communication-administration/README.md

```diff
@@ -1,6 +1,8 @@
-# Azure Communication Administration client library for JavaScript
+# Azure Communication Services Administration client library for JavaScript
```

Comment:
Do we need to add Services here? The other packages don't have it and the package name is just communication-administration